pom.xml

```diff
@@ -50,12 +50,10 @@
     <module>hawkular-bus-ra-wf-module</module>
     <module>hawkular-bus-samples</module>
     <module>hawkular-nest</module>
-    <module>hawkular-feed-comm</module>
   </modules>
 
   <properties>
     <version.org.hawkular.accounts>1.0.11.Final</version.org.hawkular.accounts>
```

Comment:
I don't think bus needs accounts either - I think this was only used for the feed comm stuff. In the dep management section I just see this:

        <artifactId>hawkular-accounts-websocket-api</artifactId>

which is the new accounts API the feed comm stuff needs.